ios/Classes/FlutterBleLibPlugin.m

```diff
@@ -178,9 +210,40 @@ - (void)dispatchEvent:(NSString * _Nonnull)name value:(id _Nonnull)value {
 
 // MARK: - Utility methods
 
-- (Resolve)jsonStringResolveForFlutterResult:(FlutterResult)result {
-    return ^(NSString *jsonString) {
-        result([JSONStringifier jsonStringFromJSONObject:jsonString]);
+- (Resolve)resolveForServicesForDevice:(FlutterResult)result {
+    return ^(NSArray *servicesArray) {
+        result([JSONStringifier jsonStringFromJSONObject:[self arrayReplacingKeys:@[@[@"id", @"serviceId"], @[@"uuid", @"serviceUuid"]] inArray:servicesArray]]);
```

Comment:
break lines after commas (readability), create an issue to refactor `arrayReplacingKeys` if needed

Reply:
Readability improved, issue created